internal/backend/fleetdb/fleetdb.go

```diff
@@ -0,0 +1,87 @@
+package fleetdb
+
+import (
+	"context"
+	"net"
+
+	"github.com/google/uuid"
+	"github.com/metal-toolbox/alloy/internal/app"
+	"github.com/metal-toolbox/alloy/internal/model"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/codes"
+
+	fleetdbapi "github.com/metal-toolbox/fleetdb/pkg/api/v1"
+)
+
+const (
+	pkgName = "internal/store"
+)
+
+// Store is an asset inventory store
+type Client struct {
+	*fleetdbapi.Client
+	logger       *logrus.Logger
+	facilityCode string
```

Comment:
The `facilityCode` var isn't in use, you can remove it

Reply:
It is used in line 41?